src/main.rs

```diff
@@ -50,10 +50,66 @@ fn main() {
     }
 }
 
+fn build_cache_without_training(cl: clap::ArgMatches) -> Result<(), Box<dyn Error>> {
+    /*! A method that enables creating the cache file without training the first model instance.
+    This is done in order to reduce building time of the cache and running the first model instance multi threaded. */
+    // We'll parse once the command line into cl and then different objects will examine it
+    let input_filename = cl.value_of("data").expect("--data expected");
+    let vw_namespace_map_filepath = Path::new(input_filename)
+        .parent()
+        .expect("Couldn't access path given by --data")
+        .join("vw_namespace_map.csv");
+    let vw: vwmap::VwNamespaceMap;
+    vw = vwmap::VwNamespaceMap::new_from_csv_filepath(vw_namespace_map_filepath)?;
+    let mut cache = cache::RecordCache::new(input_filename, cl.is_present("cache"), &vw);
```

Comment:
why pass
`cl.is_present("cache")`
instead of 'true'?

Reply:
thanks will fix